StabilityPackage.wl

```diff
@@ -730,3 +731,12 @@ sp1=ListPlot[{pts1,pts2,pts3},PlotRange->{{-30,85},{-1,1.5}},Filling-> {1-> Axis
 ImageSize-> Scaled[0.65],Frame-> False,PlotStyle->{Blue,Red,Darker[Green,0.4],Purple},PlotLegends-> {"IM|\[CapitalOmega]|","0.05 Re|\[CapitalOmega]'|","\!\(\*FractionBox[\(d\), \(dk\)]\)Re|\[CapitalOmega]'|"}]; (*The plot*)
 Return[sp1];
 ]
+
+
+(*Takes in single radius selected DO data and single radius selected elipse fit data and makes a distribution plot for a species*)
+compareDistributions[srdodata_,srelipdata_,species_]:=Module[{dopts,efpts,plot},
+dopts=Transpose@{dodat["mids"],ndensities[srdodata][[species]]//Diagonal};
+efpts=Transpose@{dodat["mids"],ndensities[srelipdata][[species]]//Diagonal};

```

Comment:
don't be afraid to use longer variable names or add more comments to explain what each is. Just a general comment, not specific to just here.

Reply:
I'm trying to add lots of comments because the variable names are getting dense even for me. Longer names is a good idea.